Snowflake.Data.Tests/IntegrationTests/SFPutGetTest.cs

```diff
@@ -408,6 +420,11 @@ private void PrepareTest(string compressionType, StageType stageType, string sta
             }
         }
 
+        private static bool IsCompressedByTheDriver()
```

Comment:
What about `ShouldCompress()` or `ShouldCompressByTheDriver`? The name `IsCompressedByTheDriver` may implicit that it has been already compressed.